pages/member/[memberSlug].js

```diff
@@ -15,7 +15,7 @@ import ThatLink from '../../components/shared/ThatLink';
 import SocialLinks from '../../components/shared/SocialLinks';
 import Icon from '../../components/shared/Icon';
 
-const DEFAULT_IMAGE = '/images/person-placeholder.jpg';
+const DEFAULT_IMAGE = 'https://that.imgix.net/members/person-placeholder.jpg';
```

Comment:
We should be using our domain, `https://images.that.tech/`
it points to the root of the bucket so all folders are assessable from it. I am pretty sure the history is that we started serving member images prior to the domain being ready. Though all is good now.

Reply:
Thanks for spotting! I copied it from where we already had it and didn't pay attention. I'll get it updated in both spots.